params.go

```diff
@@ -124,6 +124,10 @@ type Params struct {
 	// Account is deprecated form of StripeAccount that will do the same thing.
 	// Please use StripeAccount instead.
 	Account string
+
+	// This may be used to provide a new header or override an existing one on
+	// the HTTP request.
+	Headers map[string]string
```

Comment:
Thanks!

> golint actually didn't seem to catch this (probably because it's not a top-level export), but fixed anyway (and a few other things golint turned up)

Weird! It should have. Either way though, I probably should [have linked to this document on it instead](https://blog.golang.org/godoc-documenting-go-code):

> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.

Excellent though. This is good to go I think.